### PR TITLE
Write AKS deployment tutorial

### PR DESCRIPTION
**Pull request by Odonno**
Add deployment tutorial on AKS (Azure Kubernetes Service).

Requires changes included in https://github.com/surrealdb/helm-charts/pull/3

### CODE REVIEW

File: app/snippets/docs/deployment/aks/install-surrealdb-helm-chart.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$ helm repo add surrealdb https://helm.surrealdb.com
+$ helm repo update
+$ helm install \
+    --set surrealdb.path=$TIKV_URL \
+    --set surrealdb.auth=false \
+    --set ingress.enabled=false \
```
Comment (sgirones) [inline app/snippets/docs/deployment/aks/install-surrealdb-helm-chart.sh:6]: why do you use a `LoadBalancer Service` instead of an `Ingress` resource?

Comment (Odonno) [reply app/snippets/docs/deployment/aks/install-surrealdb-helm-chart.sh:6]: I tried to use an `Ingress` resource but it does not work. I never get an external IP. From what I found, I also need an `Ingress Controller` to "activate" the `Ingress`. For that to work with AKS, I need to configure a route in `Azure Application Gateway`.

IMO, using a `LoadBalancer` is way simpler as it only require a small change in the Helm chart.

File: app/snippets/docs/deployment/aks/tidb-cluster-definition.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$ curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/v1.5.1/examples/aks/tidb-cluster.yaml && \
+curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/v1.5.1/examples/aks/tidb-monitor.yaml && \
+curl -O https://raw.githubusercontent.com/pingcap/tidb-operator/v1.5.1/examples/aks/tidb-dashboard.yaml
```
Comment (sgirones) [inline app/snippets/docs/deployment/aks/tidb-cluster-definition.sh:3]: can we replace this for the actual YAML content and one similar to what I did for GKE: https://github.com/surrealdb/www.surrealdb.com/pull/354/files#diff-ac8a2424f107308101267f034ed529a7d1c7ebb93e3902b6c46668955fc24a1f

The ones provided by PingCAP require the TiDB nodepool and use too much disk for a "demo".
I think the only part that won't map to Azure is `storageClass`, but you can simply remove that line.

File: app/templates/docs/deployment/azure.hbs [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+	<blockquote Green text="Note">
+		In order to speed things up, the following commands can be executed in parallel.
+	</blockquote>
+	<ol>
+		<li>Create a TiDB Operator and monitor pool</li>
+		<Code @name="docs/deployment/aks/az-aks-nodepool-add-admin.sh" text="Create a TiDB Operator and monitor pool" />
+		<li>Create a PD node pool</li>
+		<Code @name="docs/deployment/aks/az-aks-nodepool-add-pd.sh" text="Create a PD node pool" />
+		<li>Create a TiDB node pool</li>
+		<Code @name="docs/deployment/aks/az-aks-nodepool-add-tidb.sh" text="Create a TiDB node pool" />
```
Comment (sgirones) [inline app/templates/docs/deployment/azure.hbs:62]: no need for the TiDB pool if we use a custom YAML file like in GKE

**Pull request closed without merging**